operator/controllers/external_scaler.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/kedacore/http-add-on/pkg/k8s"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
can you rename this import to `metav1`? doing so helps distinguish from `corev1`

Reply:
Done